airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -481,3 +471,43 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             Fulfillments(config),
             Shop(config),
         ]
+
+    def read(
+        self, logger: AirbyteLogger, config: Mapping[str, Any], catalog: ConfiguredAirbyteCatalog, state: MutableMapping[str, Any] = None
+    ) -> Iterator[AirbyteMessage]:
+        user_scopes = self.get_user_scopes(config)
+
+        always_permitted_streams = ["Metafields", "Shop"]
```

Comment:
Can we rename this to something like `base_streams` ?

Reply:
Metafields and Shop are not base streams. I think the `always_permitted_streams` naming better explain the main idea.